index.html

```diff
@@ -3321,8 +3321,8 @@ <h3><code>securityDefinitions</code> and <code>security</code></h3>
   for example, perhaps only those in an administrative role are authorized to perform "special" interactions.
   Tokens can have more than one scope.  
   In this example, an administrator would
-  probably be issued tokens with both the <code>limited</code> and <code>special</code> scopes,
-  while ordinary users would only be issued tokens with the <code>limited</code> scope.
+  probably issue tokens with both the <code>limited</code> and <code>special</code> scopes,
```

Comment:
"be issued" is correct in both cases.  These tokens are provided TO the administrator and the user in both cases by the authentication server.  Rather than "be issued" it might be clearer to say "be provided with".  We can also get rid of "would probably" and say "could" so: "could be provided with".  Again, for both roles.

Reply:
I interpreted the sentence the other way around. Administrators and Users will send tokens TO the remote web thing, probably my misunderstanding is due to the fact that the introduction text before the sentence refers to interactions between the client and the remote webthing:
> As another more complex example, OAuth 2.0 makes use of scopes. These are identifiers that may appear in tokens and must match with corresponding identifiers in a resource to allow access to that resource (or Interaction Affordance in the case of W3C WoT). For example, in the following, the status Property can be read by Consumers using bearer tokens containing the scope limited, but the configure Action can only be invoked with a token containing the special scope.

My suggestion is to apply your changes but also change the previous sentence from:
>  Tokens can have more than one scope
to
> Tokens can have more than one scope and are issued by dedicated services to users. 

Or something similar.

Reply:
In the device and code flows, the client is issued tokens by an authentication server, and then provides these later to the web api of the device it is trying to talk to.  The "be issued" here is referring in that case to the interaction between the client and the authentication server, not the client and the device.  So I think the confusion is we are talking about different stages of the process.

I avoided saying in this sentence "provided by the authentication server" however because only "device" and "code" flows actually use authentication servers.  The "client" flow does not (in fact disallows specifying it in the security scheme!).  Since we care mostly about the "client" flow this is important.  In the "client" flow the consumer/client would have been *provisioned* with tokens with the right scopes at some point by some other entity, i.e. during onboarding.

Anyway... the original text was correct, and was worded to cover these cases (not being overly specific about where the token came from, just that it was issued by some other entity).